ray-operator/apis/config/v1alpha1/configuration_types.go

```diff
@@ -70,6 +70,9 @@ type Configuration struct {
 
 	// DeleteRayJobAfterJobFinishes deletes the RayJob CR itself if shutdownAfterJobFinishes is set to true.
 	DeleteRayJobAfterJobFinishes bool `json:"deleteRayJobAfterJobFinishes,omitempty"`
+
+	// EnableMetrics indicates whether to collect metrics in the Ray Operator.
```

Comment:
To the best of my understanding, KubeRay emits metrics, and Prometheus collects them. I might be using the verbs incorrectly, but this is how I currently understand it.

```suggestion
	// EnableMetrics indicates whether KubeRay emits control plane metrics.
```

Reply:
Fixed. Thank you!